### PR TITLE
new icon: vitess (original, plain, original-wordmark, plain-wordmark)

### PR DESCRIPTION
**Pull request by AbdelrahmanDwedar**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

[the image used in their website source code](https://github.com/vitessio/website/blob/1a3cb3f8ac33568da8742e40d8354fa0e7daef65/wip-docs/images/vitess_logo.svg)

### CODE REVIEW

**Comment by AbdelrahmanDwedar** [comment]
I forgot what are the options to use in SVGO. When someone check this PR can you tell me the needed options?

**Comment by AbdelrahmanDwedar** [comment]
I did the optimization with SVGO but still there's a CI error. When someone checkout this PR can you tell me what did I miss?

**Comment by Snailedlt** [comment]
> I did the optimization with SVGO but still there's a CI error. When someone checkout this PR can you tell me what did I miss?

The workflow that failed is not your fault. It's because it doesn't have access to your fork, so it fails. We're looking into it :)

**Comment by AbdelrahmanDwedar** [comment]
> > I did the optimization with SVGO but still there's a CI error. When someone checkout this PR can you tell me what did I miss?
> 
> The workflow that failed is not your fault. It's because it doesn't have access to your fork, so it fails. We're looking into it :)

Oh, alright no worries 👌🏼
I'll wait till you guys solve it.

File: icons/vitess/vitess-plain.svg
Comment (lunatic-fox) [inline icons/vitess/vitess-plain.svg:1]: This icon version needs to be ungrouped and with only one path. Here's a suggestion:


Once this this icon will be used as `font` the colors will be added by `devicon.json` `color` property.

File: devicon.json [hunk 1]
```diff
@@ -9679,6 +9679,26 @@
         },
         "color": "#ffdd35",
         "aliases": []
+    },
+	{
```
Comment (lunatic-fox) [inline devicon.json:9683]:
```suggestion
    {
```

**Comment by lunatic-fox** [comment]
vitess

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
It appears there are also `wordmark` versions.
It'd be great if you could add those to the list as well. 🙂

https://cncf-branding.netlify.app/projects/vitess/

**Comment by AbdelrahmanDwedar** [comment]
I'll try to do all that stuff tomorrow, sorry I'm in a exams time so I'll not be as active.

**Comment by Snailedlt** [comment]
@AbdelrahmanDwedar Good luck with your exams if you still have some upcoming :)
Let us know if you need any help with this PR

**Comment by AbdelrahmanDwedar** [comment]
> It appears there are also `wordmark` versions. It'd be great if you could add those to the list as well. slightly_smiling_face
> 
> https://cncf-branding.netlify.app/projects/vitess/

This link breaks for me, but I found another one here
https://www.vectorlogo.zone/logos/vitessio/vitessio-ar21.svg

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
The filename for the wordmark version should be `original-wordmark`.
It'd be great if you could add `plain-wordmark` as well.

If you need any help, let us know. 🙂

**Comment by AbdelrahmanDwedar** [comment]
I think this will work.

File: devicon.json [hunk 2]
```diff
@@ -9813,6 +9813,26 @@
...
+        "altnames": [],
+        "tags": [
+            "sharding",
+            "database",
+            "mysql"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
```
Comment (Panquesito7) [inline devicon.json:9827]:
```suggestion
                "original",
                "original-wordmark",
                "plain",
                "plain-wordmark"
```

File: devicon.json [hunk 3]
```diff
@@ -9813,6 +9813,26 @@
...
+            ],
+            "font": [
+                "plain"
```
Comment (Panquesito7) [inline devicon.json:9830]:
```suggestion
                "plain",
                "plain-wordmark"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by lunatic-fox — CHANGES_REQUESTED** [review]

**Comment by lunatic-fox** [comment]
Color `#f16827` will be added by `color` property of `devicon.json`.

**Review by lunatic-fox — APPROVED** [review]
Great work! ✔
Thank you for this contribution! 🚀

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**